### DISCUSSION TITLE
apollo各服务对机器的性能有什么要求吗

### DISCUSSION THREAD

**Discussion by howHzz** [apolloconfig/apollo#3589] [Q&A] [answered]
- [ ] 我已经检查过[discussions](https://github.com/ctripcorp/apollo/discussions)
- [ ] 我已经搜索过[issues](https://github.com/ctripcorp/apollo/issues)
- [ ] 我已经仔细检查过[FAQ](https://ctripcorp.github.io/apollo/#/zh/faq/common-issues-in-deployment-and-development-phase)

**描述bug**

不是bug,想知道apollo各服务对机器的性能有什么要求吗，比如cpu 内存 磁盘的型号、快慢、大小的要求。还是普通机器就可以胜任，我们项目常用的机器就是2核4G和4核8G的

**复现**

通过如下步骤可以复现：

1. 
2. 
3. 
4. 

**期望**

简介明了地描述你希望正常情况下应该发生什么

**截图**

如果可以，附上截图来描述你的问题

### 额外的细节和日志

- 版本：
- 错误日志
- 配置：
- 平台和操作系统

**Comment by Anilople** [comment, marked as answer]
Reference https://ctripcorp.github.io/apollo/#/zh/misc/apollo-benchmark

**Reply by howHzz** [reply]
测试环境：我在2核4G的机器上部署了一个config 和一个admin，使用的默认jvm配置，admin(-Xms2560m -Xmx2560m), config(Xms6144m -Xmx6144m),但是可以正常起来和使用。

我如果生产环境用一台8c8G的机器部署一个config 和一个admin，会不会有问题，我看推荐是8C12G

**Reply by Anilople** [reply]
看有多少namespace在使用，以及namespace的大小

500个应用，每个应用4个namespace，3台 4C8G 没压力

仅供参考

---

生产不推荐单机部署，推荐集群部署，参考 https://ctripcorp.github.io/apollo/#/zh/deployment/distributed-deployment-guide
